src/components/survey/Survey.tsx

```diff
@@ -70,20 +70,30 @@ export function SurveyTimeline(props: SurveyProps) {
 
 function SurveyTimelineCard(survey: SurveyTimelineCardData) {
 
+  let getToday = new Date().toDateString();
+  const testUrl = "https://images.unsplash.com/photo-1580251645806-239f4df8ce13?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1951&q=80"
+
+  function SurveyStatusCheck(status: SurveyStatus) {
+    if (status === SurveyStatus.NEW)
+      return <hr color="green" />
+    else if (status === SurveyStatus.VIEWED)
+      return <hr color="orange" />
+    else if (status === SurveyStatus.SOLVED)
+      return <hr color="red" />
```

Comment:
### Recommendation
Of course your code will work like a charm but we can improve it like below. If you like it please let me know. 😊
```ts
export enum SurveyStatus {
  NEW = 0,
  VIEWED= 1,
  SOLVED- 2,
  ..
  ..
}
// then create a const array to declare that kind of things
// because it can grow in the future.
const hrColorStatuses = ["green", "orange", "red"];

...
...

function SurveyStatusCheck(status: SurveyStatus) {
  return <hr color={hrColorStatuses[status]} />
}
```

Reply:
Thanks for your suggestion bro. Absolutely right ı will edit :)